packages/client/src/Components/LeaderBoardList/index.tsx

```diff
@@ -0,0 +1,57 @@
+// 1. Добавить роут для лидерборда.
+// 2. Сверстать страницу лидерборда.
+// Добавлен роут для лидерборда, при переходе авторизованного пользователя на этот экран отображается список моковых рекордов.
+// Лидерборд представляет собой список рекордов пользователей вашей игры.
+
+import React from "react";
+import { Flex, List, Space, Typography } from "antd";
+
+type TLeaderBoardItem = {
+    position: number;
+    username: string;
+    score: number;
+};
+type TLeaderBoardItemProps = {
+    item: TLeaderBoardItem;
+};
+type TLeaderBoardListProps = {
+    list: TLeaderBoardItem[];
+};
+const LeaderBoardItem: React.FC<TLeaderBoardItemProps> = (props) => {
```

Comment:
'React.FC':
'Useless and has some drawbacks, see https://github.com/facebook/create-react-app/pull/8177'

Reply:
\+